mmv1/third_party/terraform/services/workstations/resource_workstations_workstation_config_test.go.erb

```diff
@@ -1293,4 +1293,91 @@ resource "google_workstations_workstation_config" "default" {
 }
 `, context)
 }
+
+func TestAccWorkstationsWorkstationConfig_vmTags(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderBetaFactories(t),
+		CheckDestroy:             testAccCheckWorkstationsWorkstationConfigDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccWorkstationsWorkstationConfig_vmTags(context),
+			},
+			{
+				ResourceName:            "google_workstations_workstation_cluster.default",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"etag"},
+			},
+		},
+	})
+}
+
+func testAccWorkstationsWorkstationConfig_vmTags(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+  data "google_project" "project" {
+    provider = "google-beta"
+  }
+  
+  resource "google_tags_tag_key" "tag_key1" {
+    provider = google-beta
+    parent = "projects/${data.google_project.project.number}"
+    short_name = "tf_test_tag_key1%{random_suffix}"
+  }
+  
+  resource "google_tags_tag_value" "tag_value1" {
+    provider = google-beta
+    parent = "tagKeys/${google_tags_tag_key.tag_key1.name}"
+    short_name = "tf_test_tag_value1%{random_suffix}"
+  }
+
+resource "google_compute_network" "default" {
+  provider                = google-beta
+  name                    = "tf-test-workstation-cluster%{random_suffix}"
+  auto_create_subnetworks = false
+}
+
+resource "google_compute_subnetwork" "default" {
+  provider      = google-beta
+  name          = "tf-test-workstation-cluster%{random_suffix}"
+  ip_cidr_range = "10.0.0.0/24"
+  region        = "us-central1"
+  network       = google_compute_network.default.name
+}
+
+resource "google_workstations_workstation_cluster" "default" {
+  provider                   = google-beta
+  workstation_cluster_id     = "tf-test-workstation-cluster%{random_suffix}"
+  network                    = google_compute_network.default.id
+  subnetwork                 = google_compute_subnetwork.default.id
+  location                   = "us-central1"
+}
+
+resource "google_workstations_workstation_config" "default" {
+  provider               = google-beta
+  workstation_config_id  = "tf-test-workstation-config%{random_suffix}"
+  workstation_cluster_id = google_workstations_workstation_cluster.default.workstation_cluster_id
+  location               = "us-central1"
+
+  host {
+    gce_instance {
+      machine_type                = "e2-standard-4"
+      boot_disk_size_gb           = 35
+      disable_public_ip_addresses = true
+      vm_tags = {
+        "tagKeys/${google_tags_tag_key.tag_key1.name}" = "tagValues/${google_tags_tag_value.tag_value1.name}"
+      }
+    }
+  }
+  
+}
+`, context)
+}
+
```

Comment:
I noticed that the new test isn't added as part of `magic-modules/mmv1/third_party/terraform/services/workstations/go/resource_workstations_workstation_config_test.go.tmpl`

could you add this? this may be the reason for the unusual errors. I don't see anything that stands out that would cause the rest of the tests to fail.

Reply:
ok. Will do.

Reply:
got confirmation that the failing tests are unrelated. We can approve and merge this with the new info 🎊